test/integ/cli/conftest.py

```diff
@@ -0,0 +1,171 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+from unittest.mock import MagicMock
+import pytest
+from deadline.job_attachments import upload
+from deadline.job_attachments._aws.deadline import get_queue
+from deadline.job_attachments.asset_manifests.hash_algorithms import HashAlgorithm, hash_file
+from deadline.job_attachments.asset_manifests.versions import ManifestVersion
+from .test_utils import JobAttachmentTest, UploadInputFilesOneAssetInCasOutputs
+
+
+@pytest.fixture(scope="session")
```

Comment:
This code is refactored from the old `sync_output` integration test. Eventually, we should delete the old sync_output test. It has no dependencies to the old test implementation so we can cleanly remove the old tests.

Reply:
This includes the test data below, once our migration is complete, we have an easy clean cut.